cmd/destroy/cmddestroy.go

```diff
@@ -127,9 +127,19 @@ func (r *DestroyRunner) RunE(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+
+	var printStatusEvents bool
+	// Print status events if a wait timeout is specified
+	if r.deleteTimeout != time.Duration(0) {
```

Comment:
This default doesn't make a lot of sense with the current default of 1min, but I wasn't sure what to change it to and it's just kapply so it's not super important.